src/typescript/views/todoView.ts

```diff
@@ -3,13 +3,17 @@ import TodoTemplate from '../templates/todoTemplate';
 
 class TodoView {
     todoListEl: HTMLElement;
+    btnSearchEl: HTMLElement;
+    inputSearchEl: HTMLInputElement;
 
     /**
      * Constructor of TodoView object
      */
 
     constructor() {
         this.todoListEl = document.querySelector('.todo-list')!;
+        this.btnSearchEl = document.querySelector('.btn-search')!;
```

Comment:
Should we define global variable? Because you use it only in one function only

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!